shared/src/business/useCases/migrateCaseInteractor.js

```diff
@@ -43,11 +43,6 @@ exports.migrateCaseInteractor = async ({
         applicationContext
           .getPersistenceGateway()
           .deleteCaseByDocketNumber({ applicationContext, docketNumber }),
-        ...caseToDelete.documents.map(({ documentId }) =>
```

Comment:
Would this end up orphaning documents?

Reply:
This was intentional per PR: https://github.com/flexion/ef-cms/pull/6333
@mmarcotte may have some better context on the reason for this.

Reply:
From @mark-meyer : 
> Yes, this was done because re-migrating cases was deleting PDFs from the S3 bucket. I think the way they made it possible to run an ‘update’ was by deleting the case and re-importing. But this had a side-effect of deleting the PDF in the bucket, which we don’t want to do when updating a migrated case. It sounds like they will need a way to distinguish between “I want to delete this case” and “I want to remigrate this case”.

Seems like we need a Trello card for this!

Reply:
Got it added. Thanks!